### ISSUE TITLE
How can I download data

### ISSUE THREAD

**Issue by rongqiibri** [perslab/bentsen-rausch-2019#1]
Hi there. I am trying to run the code to reproduce the result. The data folder is empty. Could you please share the data? Thank you

**Comment by dylanmr** [comment]
Of course! Should have put a link to the GEO series in the readme here. It can be found here: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148946

There is both fastqs - if you want to reprocess, or count matrices if you would prefer to start there. Feel free to ask any more questions if you run into any issues

**Comment by rongqiibri** [comment]
> Of course! Should have put a link to the GEO series in the readme here. It can be found here: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148946
> 
> There is both fastqs - if you want to reprocess, or count matrices if you would prefer to start there. Feel free to ask any more questions if you run into any issues

got it. Thank you so much!
